frame/support/src/storage/generator/map.rs

```diff
@@ -312,3 +318,92 @@ impl<K: FullEncode, V: FullCodec, G: StorageMap<K, V>> storage::StorageMap<K, V>
 		})
 	}
 }
+
+/// Test iterators for StorageDoubleMap
+#[cfg(test)]
+#[allow(dead_code)]
```

Comment:
What is dead code here?

Reply:
some unused metadata function, but I now added `allow(dead_code)` on those function directly.